Description: Rename control center shell to "System Settings", as per Canonical design team decision.
Bug-Ubuntu: https://launchpad.net/bugs/733234

Index: gnome-control-center-2.32.1/shell/gnomecc.desktop.in.in
===================================================================
--- gnome-control-center-2.32.1.orig/shell/gnomecc.desktop.in.in	2011-03-14 16:40:40.075236792 +0100
+++ gnome-control-center-2.32.1/shell/gnomecc.desktop.in.in	2011-03-14 16:40:51.175236792 +0100
@@ -1,5 +1,5 @@
 [Desktop Entry]
-_Name=Control Center
+_Name=System Settings
 _GenericName=The GNOME configuration tool
 Icon=gnome-control-center
 Exec=gnome-control-center
